epcis-context.jsonld

```diff
@@ -1,98 +1,328 @@
-    "minValue" : {"@type": "xsd:double"},
-    "maxValue" : {"@type": "xsd:double"},
-    "percValue" : {"@type": "xsd:double"},
-    "percRank" : {"@type": "xsd:double"},
-    "meanValue" : {"@type": "xsd:double"},
-    "sDev" : {"@type": "xsd:double"},
-    "stringValue" : {"@type": "xsd:string"},
-    "booleanValue" : {"@type": "xsd:boolean"},
-    "hexBinaryValue" : {"@type": "xsd:hexBinary"},
-    "uriValue" : {"@type": "@id"},
-    "uom": {"@type": "xsd:string"},
-    "component": {"@type": "xsd:string"},
-
-    "certificationInfo": {"@type": "@id"},
-    "epcList": {"@type": "@id"},
-    "epcClass": {"@type": "@id"},
-    "parentID": {"@type": "@id"},
-    "childEPCs": {"@type": "@id"},
-    "children": {"@type": "@id"},
-    "inputEPCList": {"@type": "@id"},
-    "outputEPCList": {"@type": "@id"},
-    "inputQuantityList": {"@type": "@id"},
-    "outputQuantityList": {"@type": "@id"},
-    "childQuantityList": {"@type": "@id"},
-
-    "bizStep": {"@type": "@id"},
-    "disposition": {"@type": "@id"},
-    "persistentDisposition": {"@type": "@id"},
-    "bizTransactionList": {
-      "@type": "@id",
-      "@context": {"type": {"@id":"epcis:bizTransactionType", "@type":"@id"}}},
-    "bizTransaction": {"@type": "@id"},
-    "bizLocation": {"@type": "@id"},
-    "readPoint": {"@type": "@id"},
-    "source": {"@type": "@id"},
-    "sourceList": {
-      "@type": "@id",
-      "@context": {"type": {"@id":"epcis:sourceOrDestinationType", "@type":"@id"}}},
-    "destination": {"@type": "@id"},
-    "destinationList": {
-      "@type": "@id",
-      "@context": {"type": {"@id":"epcis:sourceOrDestinationType", "@type":"@id"}}},
-
-    "reason": {"@type": "@id"},
-    "errorDeclaration": {"@type": "@id"},
-    "correctiveEventIDs": {"@type": "@id"},
-    
-    "set": {"@type": "@id"},
-    "unset": {"@type": "@id"},
-    "sensorElementList": {"@type": "@id"},
-    "sensorMetadata": {"@type": "@id"},
-    "sensorReport": {
-      "@type": "@id",
-      "@context": {"type": {"@id":"epcis:measurementType", "@type":"@id"}}},               
-    "deviceID": {"@type": "@id"},
-    "deviceMetadata": {"@type": "@id"},
-    "rawData": {"@type": "@id"},
-    "bizRules": {"@type": "@id"},
-    "dataProcessingMethod": {"@type": "@id"},
-    "chemicalSubstance": {"@type": "@id"},
-    "microorganism": {"@type": "@id"},
-        
-    "isA": "@type",
-    "id": "@id",
-    "eventID": "@id",
-    "creationDate": {"@id": "dcterms:created", "@type": "xsd:dateTime"},
-    "schemaVersion": {"@id": "owl:versionInfo", "@type": "xsd:string"}
-  }
+    "@context": {
+        "@protected": true,
+        "@version": 1.1,
+        "epcis": "https://ns.gs1.org/epcis/",
+        "gs1": "https://gs1.org/voc/",
+        "owl": "http://www.w3.org/2002/07/owl#",
+        "xsd": "http://www.w3.org/2001/XMLSchema#",
+        "dcterms": "http://purl.org/dc/terms/",
+        "id": "@id",
+        "isA": "@type",
+        "ObjectEvent": "epcis:ObjectEvent",
```

Comment:
for the same reason as avoiding `@base` because of the unintended effects of `@vocab`.  If you specify `@vocab` then for any  non qualified term used in a document  it will be expanded to `epcis:xxx` which it is totally incorrect. 

That's why  if you take a look no @context out there intended to production-ready work uses @vocab that way.

Reply:
You believe that `@vocab` is somehow harmful, but I disagree with you. Here are some examples: important ontologies, or those made by key JSONLD members:
- schema.org uses `@vocab: "http://schema.org/"`, see http://schema.org/docs/jsonldcontext.jsonld
- WoT TD uses `@vocab` at top level, and also nested `@vocab` in nested contexts, see https://www.w3.org/2019/wot/td/v1
- CSVW uses nested `@type:@vocab`, see `curl -Haccept:application/ld+json http://www.w3.org/ns/csvw`
- Hydra uses nested `@type:@vocab`, see http://www.w3.org/ns/hydra/context.jsonld
- IIIF uses nested `@type:@vocab`, see https://iiif.io/api/presentation/3/context.json

> If you specify @vocab then for any non qualified term used in a document it will be expanded to epcis:xxx which it is totally incorrect

If the user types `"epcis:xxx"` you'll get the same effect.
Do you also want to remove the `epcis:` prefix from the context??